### ISSUE TITLE
Can we use it on streaming?

### ISSUE THREAD

**Issue by ghostzero94042** [NickCis/among-us-proxy#9]
Official server have two problem:
1.high ping
2.usually disconnect
and some streamers play together need stable game room, and they will get some donate or subscription. can we use this project on among us stream and get donate or subscription?

**Comment by NickCis** [comment]
Yeah, you could use it. As the readme describes, this allow to proxy the lan multiplayer, so, if everybody gets the client, they will be able to connect.

If you want, you could describe a little bit more how the perfect way would work and we can figure out what is it possible to do.
